src/DotNetRuntimeValidation/Directory.Build.targets

```diff
@@ -0,0 +1,27 @@
+<Project>
+
+  <Import Project="..\..\Directory.Build.targets" />
+
+  <PropertyGroup>
+    <TempZipFolder>$(ArtifactsBinDir)DotNetRuntimeValidation\$(TargetArchitecture)\temp\zipcontents</TempZipFolder>
+  </PropertyGroup>
+
+  <!-- We just want to .zip these up -->
+  <Target Name="_ZipValidationExe" AfterTargets="Build">
```

Comment:
Would this target work fine in incremental build?

Reply:
I should probably add conditions on the the .zip not already existing

Reply:
Updated